account_mass_reconcile_partner/models/mass_reconcile.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import models, api
+
+
+class AccountMassReconcileMethod(models.Model):
+    _inherit = 'account.mass.reconcile.method'
+
+    @api.model
+    def _get_all_rec_method(self):
+        res = super(AccountMassReconcileMethod, self)._get_all_rec_method()
+        res.append((
+            'mass.reconcile.advanced.partner',
+            'Advanced. Partner'
```

Comment:
"Advanced. Partner" correct w/ that dot?

Reply:
Yes, it's to stay consistent with the existing : https://github.com/OCA/account-reconcile/blob/11.0/account_mass_reconcile/models/mass_reconcile.py#L81